docroot/modules/custom/sitenow_p2lb/src/Commands/P2LbCommands.php

```diff
@@ -158,6 +158,18 @@ public function cleanup(array $options = ['batch' => 5]) {
     $config_import->run($config_import->showRealtime());
     $config_import->getOutput();
     drupal_flush_all_caches();
+    // Ensure 'page' is in the replicate allowed content types.
+    $uiowa_core_settings = $this->configFactory->getEditable('uiowa_core.settings');
```

Comment:
```suggestion
    $uiowa_core_settings = $this->configFactory
      ->getEditable('uiowa_core.settings');
```
Usually these chained methods are put on new lines to make them more readible.